### DISCUSSION TITLE
*Update* Any New Websites like sites.google or something that the people can actually use.

### DISCUSSION THREAD

**Discussion by winghf** [3kh0/ext-remover#445] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
For the people and me who want some websites can be any like github or sites.google (for me .rep or something like that dosnt work for me its blocked)

**Comment by obkduishinijf** [comment, marked as answer]
https://aadynpersonalunblocker.glitch.me
simple little ub, not mine.
